server/webapp/WEB-INF/rails.new/spec/new_javascripts/pipeline_configs/views/test_connection_widget_spec.js

```diff
@@ -46,7 +46,7 @@ define(["mithril", "lodash", "pipeline_configs/models/materials", "pipeline_conf
 
         mount(material, {connectionState: state});
 
-        expect($root.find(".alert-box")[0].innerText).toBe('Test Connection Failed');
+        expect($($root.find(".alert-box")[0]).text()).toBe('Test Connection Failed');
```

Comment:
See https://github.com/velesin/jasmine-jquery for how to write improved assertions.